clang/test/SemaSYCL/loop_unroll.cpp

```diff
@@ -32,11 +32,6 @@ void foo() {
   [[clang::loop_unroll(4)]]
   for (int i = 0; i < 10; ++i);
 
-  // expected-error@+2 {{incompatible loop unroll instructions: '#pragma unroll(4)' and '[[clang::loop_unroll(2)]]'}}
-#pragma unroll 4
-  [[clang::loop_unroll(2)]]
-  for (int i = 0; i < 10; ++i);
-
```

Comment:
Why? It seems like valid diagnostic test.

Reply:
Okay, I've read this explanation https://github.com/intel/llvm/issues/1893#issuecomment-644326152, but the commit message is a bit confusing.

Reply:
see analysis from Viktoria in https://github.com/intel/llvm/issues/1893